src/test/scala/chiselTests/ProbeSpec.scala

```diff
@@ -225,7 +258,20 @@ class ProbeSpec extends ChiselFlatSpec with Utils {
     )
   }
 
-  ":= connector with aggregate of probe" should "fail" in {
+  ":= connector with probe" should "work" in {
+    val chirrtl = ChiselStage.emitCHIRRTL(
+      new RawModule {
+        val io = IO(new Bundle {
+          val in = Input(Probe(Bool()))
+          val out = Output(Probe(Bool()))
+        })
+        io.out := io.in
+      }
+    )
+    processChirrtl(chirrtl) should contain("define io.out = io.in")
```

Comment:
can you add a case for the same as the above example, but with user error of `io.in := io.out` ? presumably we get an error message, not sure if it is in Chisel -> .fir or .fir -> .v?